### ISSUE TITLE
Does not detect GPU

### ISSUE THREAD

**Issue by jsissom** [Pavelevich/llm-checker#1]
```
llm-checker ai-check
✔ ✅ AI selection completed!

 🧠 INTELLIGENT MODEL SELECTION
╭─────────────────────────────────────────────────────────────────
│ 🏆 Selected Model: phi4:latest
│ 🎯 Selection Method: INTELLIGENT MATHEMATICAL
│ 📊 Confidence: 99%
│ 🔢 Intelligence Score: 90/100
│ 💡 AI Analysis: Excellent fit for your medium hardware configuration. Optimal memory utilization. CPU well-suited for this model.
╰

 💻 INTELLIGENT HARDWARE ANALYSIS
╭───────────────────────────────────────────────────────
│ CPU: 8 cores @ 3.0 GHz
│ RAM: 15.0 GB
│ GPU: cpu_only
│ VRAM: 0.0 GB
│
│ Hardware Classification:
│   Overall Tier: MEDIUM
│   Available Memory: 10.5 GB
│   Performance Index: ×1.0
╰

 🎯 RECOMMENDATION
╭──────────────────────────────────────────────────
│ Best model for your hardware:
│   ollama run phi4:latest
│
│ Why this model?
│   • Optimized for your hardware configuration
│   • Confidence: 99%
│   • Selection method: INTELLIGENT_MATHEMATICAL


💡 Tips:
  • To execute the recommended model directly:
    llm-checker ai-run
  • To exit from Ollama chat, type: /bye


Here are some commands from the system showing that it has a GPU:

 lspci | grep -i nvidia
00:10.0 VGA compatible controller: NVIDIA Corporation Device 2d04 (rev a1)
00:10.1 Audio device: NVIDIA Corporation Device 22eb (rev a1)

nvidia-smi
Tue Aug  5 12:05:20 2025
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 575.64.03              Driver Version: 575.64.03      CUDA Version: 12.9     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA GeForce RTX 5060 Ti     Off |   00000000:00:10.0 Off |                  N/A |
|  0%   28C    P8              4W /  180W |      34MiB /  16311MiB |      0%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+

+-----------------------------------------------------------------------------------------+
| Processes:                                                                              |
|  GPU   GI   CI              PID   Type   Process name                        GPU Memory |
|        ID   ID                                                               Usage      |
|=========================================================================================|
|    0   N/A  N/A            5964      G   /usr/lib/xorg/Xorg                        4MiB |
+-----------------------------------------------------------------------------------------+

**Comment by Pavelevich** [comment]
hi, please try the new version: npm i llm-checker@2.2.0 the problem with GPU detection was resolved

<img width="1093" height="504" alt="Image" src="https://github.com/user-attachments/assets/4873dbf6-99b9-42a2-bd11-8f15bd3dd400" />

**Comment by Pavelevich** [comment]
hi, New version 2.2.2: The fix I made allows systeminformation to use the device ID as a fallback to correctly identify the GPU when it can't resolve the full name. You can try again; the VRAM size should be visible.

**Comment by jsissom** [comment]
My card is "PNY - GeForce RTX 5060 Ti Overclocked 16GB GDDR7 PCI Express 5.0 Graphics Card with Dual Fan - Black"

**Comment by Pavelevich** [comment]
hallo, there is a new version, maybe can u try it: https://www.reddit.com/r/ollama/comments/1moy27m/finally_released_the_major_update_ive_been/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button
